Manual/manJHUGenerator.tex

```diff
@@ -1351,6 +1351,8 @@ \section{JHU Generator Coupling Converter (JHUGenLexicon)}
 useMCFMAtInput=true or useMCFMAtOutput=true. useMCFMAtInput=True assumes that the $g^VV$ couplings are divided by 2. 
 useMCFMAtOutput=True divides all of the $g^{VV}$ couplings by 2 as per MCFM conventions.
 
+As described earlier, the Lagrange density function is dependent on the sign convention of the covariant derivative used to derive it. This also makes the translations dependent on the sign convention used in the covariant derivative. To switch the sign convention on the covariant derivative from $D_\mu = \partial_\mu -\mathrm{i} \frac{e}{2 s_w} \sigma^i W_\mu^i + \mathrm{i} \frac{e}{2 c_w} B_\mu$ to $D_\mu = \partial_\mu -\mathrm{i} \frac{e}{2 s_w} \sigma^i W_\mu^i - \mathrm{i} \frac{e}{2 c_w} B_\mu$ use the input flag switch\_convention = true. This effectively acts as an overall sign switch on every $Z\gamma$ coupling.
```

Comment:
Lagrange density is dependent -> the Lagrangian depends on?

Reply:
Also relevant here is how you distinguish the option from regular text. That also needs a different style as mentioned for the other comment.